datafusion/sqllogictest/test_files/interval.slt

```diff
@@ -25,27 +25,10 @@ select
 Interval(MonthDayNano) Interval(MonthDayNano)
 
 
-## This is incredibly confusing but document it in tests:
-#
-# years is parsed as a column name
-# year is parsed as part of the interval type.
-#
-# postgres=# select interval '5' year;
-#  interval
-# ----------
-#  5 years
-# (1 row)
-#
-# postgres=# select interval '5' years;
-#   years
-# ----------
-#  00:00:05
-# (1 row)
 query ?
 select interval '5' years
 ----
-5.000000000 secs
-
+60 mons
```

Comment:
this certainly seems much more correct to me -- I also removed the now out of date comment.